src/components/Editor.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render, screen } from "@testing-library/react";
+import { Design } from "react-email-editor";
+import { AppServices } from "../abstractions";
+import { AppServicesProvider } from "./AppServicesContext";
+import { Editor } from "./Editor";
+
+const emailEditorPropsTestId = "EmailEditor_props";
+
+const sampleDesign: Design = {
+  counters: {},
+  body: {
+    rows: [],
+  },
+};
+
+const unlayerProjectId = 12345;
+const unlayerEditorManifestUrl = "unlayerEditorManifestUrl";
+const loaderUrl = "loaderUrl";
+const unlayerUserId = "unlayerUserId";
+const unlayerUserSignature = "unlayerUserSignature";
+
+const authenticatedSession = {
+  status: "authenticated",
+  unlayerUser: {
+    id: unlayerUserId,
+    signature: unlayerUserSignature,
+  },
+};
+
+describe(Editor.name, () => {
+  it("should render EmailEditor with the right props when the session is authenticated", async () => {
```

Comment:
Well, my tests are not sooo useful, but at least verify if part of the props that we send to the EmailEditor component is fine.

@Cromeror could you review it?